Frends.Facebook.Request/Frends.Facebook.Request/Frends.Facebook.Request.cs

```diff
@@ -0,0 +1,159 @@
+﻿#pragma warning disable SA1200 //Using directives should be placed correctly.
+
+using Frends.Facebook.Request.Definitions;
+using Frends.HTTP.Request.Definitions;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Frends.Facebook.Request;
+
+/// <summary>
+/// Facebook class.
+/// </summary>
+public static class Facebook
+{
+    /// <summary>
+    /// This is task for reading data from Facebook API.
+    /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.Facebook.Get).
+    /// </summary>
+    /// <param name="input">Set reference type, parameters and token.</param>
+    /// <param name="options">Optional parameters.</param>
+    /// <param name="cancellationToken">Cancellation token given by Frends.</param>
+    /// <returns>Object { bool Success, dynamic Message }.</returns>
+    public static async Task<Result> Request([PropertyTab] Input input, [PropertyTab] Options options, CancellationToken cancellationToken)
+    {
+        if (string.IsNullOrEmpty(input.AccessToken))
+        {
+            throw new ArgumentNullException(nameof(input.AccessToken) + " cannot be empty.");
+        }
+        else if (string.IsNullOrEmpty(input.ApiVersion))
+        {
+            throw new ArgumentNullException(nameof(input.ApiVersion) + " cannot be empty.");
+        }
+        else if (string.IsNullOrEmpty(input.Reference))
+        {
+            throw new ArgumentNullException(nameof(input.Reference) + " cannot be empty.");
+        }
+        else if (string.IsNullOrEmpty(input.Message) && Enum.GetNames(typeof(SendMethods)).Contains(input.Method.ToString()))
+        {
+            throw new ArgumentNullException(nameof(input.Message) + " cannot be empty.");
+        }
+
+        var headers = GetHeaderDictionary(input);
+
+        using var content = GetContent(input, headers);
```

Comment:
Not sure if this new using statement is supported on net471

Reply:
Did you test the Task in net471 on a legacy agent?

Reply:
The task has been tested in both net471 and net 6.0
Using statement is ignored in net471, for I have added .dispose() -elements.